w4/protocol/api/controller/merchant_controller.go

```diff
@@ -67,3 +70,45 @@ func (c *merchantController) Create(ctx *fiber.Ctx) error {
 		"data":    response,
 	})
 }
+
+func (c *merchantController) BrowseNearby(ctx *fiber.Ctx) error {
+	context := ctx.Locals("ctx").(context.Context)
+	tracer := ctx.Locals("tracer").(trace.Tracer)
+
+	_, span := tracer.Start(context, "Browse")
+	defer span.End()
+	coordinate := strings.Split(ctx.Params("latlon"), ",")
+	coordinate[0] = strings.TrimSpace(coordinate[0])
+	coordinate[1] = strings.TrimSpace(coordinate[1])
+
+	lat, err := strconv.ParseFloat(coordinate[0], 64)
+	if err != nil {
+		return fiber.ErrBadRequest
+	}
+
+	lon, err := strconv.ParseFloat(coordinate[1], 64)
+	if err != nil {
+		return fiber.ErrBadRequest
+	}
+
+	log.Println(lat)
+	log.Println(lon)
+	var query usecase.MerchantFetchQuery
+
+	if err := ctx.QueryParser(&query); err != nil {
+		return fiber.ErrBadRequest
+	}
```

Comment:
1. If the query param's value doesn't match with the requirement, then return empty array as the results